include/octo-logger-cpp/context-info.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file context-info.hpp
+ * @author arad yaron (aradyaron98@gmail.com)
+ * @brief
+ * @version 0.1
+ * @date 2024-04-25
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+
+#ifndef CONTEXT_INFO_HPP_
+#define CONTEXT_INFO_HPP_
+
+#include <unordered_map>
+#include <string>
+#include <string_view>
+#include <sstream>
+
+
+namespace octo::logger
+{
+
+class ContextInfo
+{
+public:
+    typedef std::string_view ContextInfoKey;
```

Comment:
why not be explicit on types as those are std types?

Reply:
@SheyDenis request, I don't care wither way